Ms/default content lang

micstra opened the pull request:
### Default content lang configuration

Currently the content( i18n )-language is always set to English. This PR makes the content-language configurable, to allow other-languages as default-lang. The ideal solution would be to have the content-language user-configurable, but a config-option for a default-lang would be a quick win with little effort.

### Detailed description

- Adds a new configuration-fied `defaultContentLanguage`. 
- Exposes the field in the config-service. 
- Remove static-set of 'en' as i18n-language.
- Use config-field as initial value for i18n.
- Use config-field as initial value for the language-switcher.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/gentics/mesh-ui?pullRequest=336) <br/>All committers have signed the CLA.

micstra committed:
Add default-content-language as configuration option.

Currently the i18n of the application is always set to English. This should be configurable.

micstra committed:
Derive i18n-language from the configuration

micstra committed:
Adapt config- and i18n-service-mocks to comply with the new DEFAULT_CONTENT_LANGUAGE field

micstra committed:
Remove force-set of English-i18n

The default i18n-language should be derived from the configuration.

Jotschi commented:
@micstra Thanks for the PR. We have unfortunately no resources to review the PR at the moment. I'll let you know when / if this changes.
